analysis/import/CSVImporter/src/main/kotlin/de/maibornwolff/codecharta/importer/util/ParserDialogHelper.kt

```diff
@@ -0,0 +1,30 @@
+package de.maibornwolff.codecharta.importer.util
+
+import com.github.kinquirer.KInquirer
+import com.github.kinquirer.components.promptInput
+import java.io.File
+import java.nio.file.Paths
+
+class ParserDialogHelper {
+
+    companion object {
+        fun getInputFiles(isSourceMonitor: Boolean): MutableList<String> {
+            val inputFileNames = mutableListOf(KInquirer.promptInput(
+                    message = if (isSourceMonitor) { "What is the sourcemonitor CSV file that has to be parsed?" } else { "Please specify the name of the first sourcemonitor CSV file to be parsed" },
+                    hint = Paths.get("").toAbsolutePath().toString() + File.separator + "yourInput.csv"))
+
+            while (true) {
+                val additionalFile = KInquirer.promptInput(
+                        message = "If you want to parse additional sourcemonitor CSV files, specify the name of the next file. Otherwise, leave this field empty to skip.",
```

Comment:
Here the if condition for type sourcemonitor is missing.

Reply:
As in my other comment, previously we asked this question for the CSVImporter and the CSVSourceMonitorImporter. I can adjust this so it is neutral, and rather ask for csv files so it fits both or ask an unique question for both. What do you think?

Reply:
I like that idea!